lib/platform/github/index.spec.ts

```diff
@@ -1105,8 +1107,10 @@ describe(getName(), () => {
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
     it('creates issue if not ensuring only once', async () => {
-      httpMock
-        .scope(githubApiHost)
+      const scope = httpMock.scope(githubApiHost);
+      initRepoMock(scope, 'some/repo');
```

Comment:
Wouldn't it be easier to pass `githubApiHost` to `initRepoMock` and return the scope from it. I think this will make code more readable.

Reply:
Too much changes for this PR, as we should also refactor similar mock function for `initPlatform()` deciding on which of them should return scope and which should receive it from argument